app/components/GettingStartedModal/steps/WelcomeStep.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { Logo } from '../../basic';
+import StageUtils from '../../../utils/stageUtils';
+import LogoLabel from '../../LogoLabel';
+
+const t = StageUtils.getT('gettingStartedModal');
+
+const WelcomeStep = () => {
+    const theme = useContext(ThemeContext);
+
+    return (
+        <div style={{ backgroundColor: theme.mainColor, textAlign: 'center', margin: -19, padding: '9em' }}>
```

Comment:
I'm wondering maybe `MessageContainer` will work here instead of customized div?

Reply:
Unfortunately it would require much more style overrides, so I'd leave `div` here.

Reply:
Oh, my bad, I had `FullScreenSegment` in mind, not `MessageContainer`.

We can have:
```suggestion
        <FullScreenSegment style={{ textAlign: 'center', margin: -19, padding: '9em' }}>
```

Anyways, treat it as optional.

Reply:
I tried but it didn't work the same way - looks like more style overrides would be needed, so I gave it up.